src/test/java/io/personium/jersey/engine/test/ScriptTestBase.java

```diff
@@ -544,4 +552,17 @@ protected static String getVersion() {
         return PersoniumEngineTestConfig.getVersion();
     }
 
+    /**
+     * Function for getting base Url
+     * @return URL which Grizzly starts listening
+     */
+    @Override
+    protected URI getBaseURI() {
+        try {
+            return new URI(this.jerseyBaseUrl);
+        } catch(URISyntaxException e) {
+            e.printStackTrace();
+            return null;
```

Comment:
Can you throw an RutimeException like IllegalArgumentException instead of null?

```java
} catch (URISyntaxException e) {
    throw new IllegalArgumentException(e.message, e);
}
```